lisk-dex-electron/src/App.jsx

```diff
@@ -605,63 +603,70 @@ class App extends React.Component {
-          </div>
-          <div className="sell-panel">
-            <PlaceOrder side="ask" orderSubmit={this.orderSubmit} orderSubmitError={this.orderSubmitError}></PlaceOrder>
-          </div>
-          <div className="buy-panel">
-            <PlaceOrder side="bid" orderSubmit={this.orderSubmit} orderSubmitError={this.orderSubmitError}></PlaceOrder>
+    return (
+      <>
+        <userContext.Provider value={{ ...this.state }}>
+          {this.state.displaySigninModal && <SignInModal failure={this.state.signInFailure} passphraseSubmit={this.passphraseSubmit} enabledAssets={this.state.enabledAssets} close={this.closeSignInModal} walletGenerated={this.walletGenerated} />}
+          {this.state.displayLeaveWarning && <LeaveWarning setDisplayLeaveWarning={this.setDisplayLeaveWarning} />}
+          <div className="top-bar">
+            <div>
+              <b style={{ fontSize: '21px' }}>{this.state.configuration.appTitle}</b>
+              {' '}
+&nbsp;
+              <a className="feedback-link" style={{ color: '#34cfeb', fontSize: '14px' }} href={this.state.configuration.feedbackLink.url} rel="noopener noreferrer" target="_blank">{this.state.configuration.feedbackLink.text}</a>
+            </div>
+            <div>
+              <SignInState className="sign-in-state" showSignIn={this.showSignIn} keys={this.state.keys} signedIn={this.state.signedIn} signOut={this.signOut} />
+            </div>
           </div>
-          <div className="orderbook-container">
-            <div className="sell-orders">
-              <Orderbook side="asks" orderBookData={this.state.orderBookData} priceDecimalPrecision={this.state.priceDecimalPrecision}></Orderbook>
+          <div className="container">
+            <div className="notifications">
+              {this.state.notifications.map((data) => <Notification key={data.id} data={data} />)}
             </div>
-            <div className="price-display">
-              Price: {this.state.minAsk} {this.state.activeAssets[1].toUpperCase()}
+            <div className="sell-panel">
+              <PlaceOrder side="ask" orderSubmit={this.orderSubmit} orderSubmitError={this.orderSubmitError} />
             </div>
-            <div className="buy-orders">
-              <Orderbook side="bids" orderBookData={this.state.orderBookData} priceDecimalPrecision={this.state.priceDecimalPrecision}></Orderbook>
+            <div className="buy-panel">
+              <PlaceOrder side="bid" orderSubmit={this.orderSubmit} orderSubmitError={this.orderSubmitError} />
+            </div>
+            <div className="orderbook-container">
+              <div className="sell-orders">
+                <Orderbook side="asks" orderBookData={this.state.orderBookData} priceDecimalPrecision={this.state.priceDecimalPrecision} />
+              </div>
+              <div className="price-display">
+                Price:
+                {' '}
```

Comment:
This is strange. Can we disable the rule which causes this? `{' '}` - Also below on line 642 and other parts of the code.

Reply:
I think if space is added without JSX, it will not be rendered properly. As you can see, in the previous code, after price there is an empty space... linter automatically converts it to JSX based spacing.